src/Blocks/components/country/components/country-options.js

```diff
@@ -36,9 +39,13 @@ export const CountryOptions = (attributes) => {
 						label={<NameFieldLabel value={countryName} />}
 						help={__('Identifies the field within form submission data. Must be unique.', 'eightshift-forms')}
 						value={countryName}
-						onChange={(value) => setAttributes({ [getAttrKey('countryName', attributes, manifest)]: value })}
+						onChange={(value) => {
+							setIsNameChanged(true);
+							setAttributes({ [getAttrKey('countryName', attributes, manifest)]: value });
+						}}
 						disabled={isOptionDisabled(getAttrKey('countryName', attributes, manifest), countryDisabledOptions)}
```

Comment:
Can potentially extract `getAttrKey('countryName', attributes, manifest)`

Reply:
tnx but there is no big gain here so I will not do it tnx